core-feature-pack/legacy-feature-pack/pom.xml

```diff
@@ -0,0 +1,408 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly.core</groupId>
+        <artifactId>wildfly-core-feature-pack-parent</artifactId>
+        <version>13.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>wildfly-core-feature-pack</artifactId>
```

Comment:
should we rename the artifactId to `wildfly-core-legacy-feature-pack` and accept that starting from now a feature pack means Galleon unless specified otherwise?

Reply:
(although it is not required, I like to have Maven GAV matching the names of the directories as much as possible)

Reply:
We'd need to discuss with the keycloak folks to see if it breaks them.

Reply:
never mind, that's a silly suggestion. This artifact is consumed by other project than wildFly and it is not correct to change its GAV just for the sake of changing it.

Reply:
If we can eventually reach a point where the WildFly galleon FPs no longer depend on the wildfly-core FP, then the GAV of the wildfly-core FP becomes much less importance as most (maybe all) of its use outside core's own build will be gone.